packages/tools/pluggable-widgets-tools/configs/webpack.native.config.js

```diff
@@ -24,7 +24,12 @@ const widgetConfig = {
         rules: [
             {
                 test: /\.tsx?$/,
-                loader: "ts-loader"
+                loader: "ts-loader",
+                options: {
+                    compilerOptions: {
+                        target: "esnext"
```

Comment:
You already set it to `esnext` in tsconfig, so this i think is not needed

Reply:
That's the module. Target in tsconfig is es5